Dockerfile-73

```diff
@@ -133,8 +135,16 @@ RUN cp /etc/php/7.0/mods-available/* $CONF_PHPMODS && \
     phpenmod overrides && \
     # Enable NewRelic via Ubuntu symlinks, but disable in file. Cross-variant startup script uncomments with env vars.
     phpenmod newrelic && \
-    # - Run standard set of tweaks to ensure runs performant, reliably, and consistent between variants
-    /bin/bash -e /prep-php.sh
+    # Run standard set of tweaks to ensure runs performant, reliably, and consistent between variants
+    /bin/bash -e /prep-php.sh && \
+    # Add new 7.3+ conf
+    sed -i "s/;decorate_workers_output.*/decorate_workers_output = no/" $CONF_FPMPOOL && \
```

Comment:
is `decorate_workers_output = no` backwards compatible? or is it necessary _to be_ backwards compatible?

Reply:
no, it only exists in 7.3+